program/rust/src/processor/add_price.rs

```diff
@@ -48,10 +50,12 @@ pub fn add_price(
     )?;
 
 
-    let (funding_account, product_account, price_account, permissions_account) = match accounts {
-        [x, y, z, p] => Ok((x, y, z, p)),
-        _ => Err(OracleError::InvalidNumberOfAccounts),
-    }?;
+    let (funding_account, product_account, price_account, permissions_account, scores_account) =
+        match accounts {
+            [x, y, z, p] => Ok((x, y, z, p, None)),
+            [x, y, z, p, s] => Ok((x, y, z, p, Some(s))),
```

Comment:
This part is tricky because:
a. Does it go out of sync if we don't pass the account
b. How does the program know s is _the_ score account, or can there be many score accounts?

Reply:
Maybe it's ok since this is a permissioned operation on a second thought

Reply:
It is also just for backward compatibility and will remove the first clause when proposals have been migrated.